pkg/v3/flows/conditional.go

```diff
@@ -57,56 +60,33 @@ func NewConditionalEligibility(
 	}
 
 	return &ConditionalEligibility{
+		mStore:  ms,
 		builder: builder,
 		final:   point,
 		logger:  logger,
 	}, []service.Recoverable{svc0, svc1}, err
 }
 
 func (flow *ConditionalEligibility) ProcessOutcome(outcome ocr2keepersv3.AutomationOutcome) error {
-	var ok bool
+	samples, err := ocr2keepersv3.UpkeepIdentifiersFromOutcome(outcome)
```

Comment:
I put it where it is because I thought the scope of the function was a general helper function for accessing data in an outcome. While `flows` is the only package that currently uses it, I would prefer it be available for all because the scope isn't special for `flows`.

Reply:
Maybe it should be a static function of `AutomationOutcome`?